### DISCUSSION TITLE
Directus 10 - Autostart Node

### DISCUSSION THREAD

**Discussion by dearmosin** [directus/directus#18509] [Archive]
### Subject Area

Unable to Locate start.js In Directus 10

### Suggestions

I am having a hard time finding the location of start.js in the new Directus 10 folder structure. I want to locate start.js which was earlier in Directus 9 located at node_modules/directus/dist/start.js

I want to autostart or keep alive Directus 10 after the server reboot.

**Comment by rijkvanzanten** [comment]
This is not something we'll document, as that's not an officially recommended nor supported way of starting the server. Use the `npx directus start` command instead
